Counties/Florida/Bay County/Scraper/tests/test_Scrape.py

```diff
@@ -0,0 +1,39 @@
+import csv
+import pytest
+import unittest
+import Scraper
+
+@pytest.fixture(scope='module')
+def testdatadir(request):
+    # Get a py.path.local, which is a bit friendlier to work with.
+    return request.fspath.join('..').join('testdata')
+
+def csv2dictlist(csvfile):
+    with open(csvfile, 'r') as w:
+        dl = [r for r in csv.DictReader(w)]
+        for d in dl:
+            del d["_id"]  # this is random, so strip it for comparisons
+        return dl
+
+class TestScrape:
+
+    def test_scrape(self, tmpdir, testdatadir):
+        td = tmpdir.mkdir('scraper')
+        outfile = td.join('bay-county-scraped.csv')
+        wantfile = testdatadir.join('bay-county-scraped.csv.golden')
+
+        # TODO(mcsaucy): refactor Scraper to use less global state.
+        Scraper.settings['max-records'] = 3
+        Scraper.settings['solve-captchas'] = True
```

Comment:
It seems for this test we have to go against the advice on solving captchas. I think we might have to get extra guidance for this one.